### PR TITLE
v2.11.0 - apps promo banner

### PR DESCRIPTION
**Pull request by dandel10n**
### Added
- `c-appsBanner` component

### Changed
- `fozzie` module version update

In future for mobile there will be only one "download the app" button depending on what device you are + image will be different.
[Image from Gyazo](https://gyazo.com/8457d05568fa6110f1bc089e589283a8)

## UI Review Checks

- [x] This PR has been checked with regard to our brand guidelines
- [x] This code has been checked with regard to our accessibility standards

## Browsers Tested

- [x] Chrome
- [x] Safari
- [x] Firefox
- [x] IE 11
- [x] Edge
- [x] Mobile (i.e. iPhone/Android - via responsive browser mode)

### CODE REVIEW

File: docs/src/templates/includes/components/core/je-appsBanner.hbs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="c-appsBanner">
+    <div class="c-appsBanner-image">
```
Comment (ashleynolan) [inline docs/src/templates/includes/components/core/je-appsBanner.hbs]: Bit confusing to use `c-appsBanner-image` and `c-appsBanner-img`.  Probably best off calling this one `c-appsBanner-imgWrapper` and keeping the other one as `c-appsBanner-img`

Comment (kevinrodrigues) [reply docs/src/templates/includes/components/core/je-appsBanner.hbs]: Would it be better to just call it `c-appsBanner-wrapper` and `c-appsBanner` (leave of the `img` part) as they're used on images anyway so probably won't need that naming convention?

Comment (dandel10n) [reply docs/src/templates/includes/components/core/je-appsBanner.hbs]: @ashleynolan actually I don't use `c-appsBanner-img` in css so would it be enough to just leave c-appsBanner-image?

Comment (dandel10n) [reply docs/src/templates/includes/components/core/je-appsBanner.hbs]: I've removed `c-appsBanner-img` class, so now it shouldn't be confusing :)

**Pull request merged**